docs/README.md

```diff
@@ -4,9 +4,7 @@
   - Creating a new application with the US Forms System Starter App
   - Installing the library in an existing application
 - [Building a form](building-a-form/README.md)
-  - About the JSON schema standard and react-jsonschema-form
-  - Adding widgets
-  - Adding definitions
-  - Creating custom widgets
-  - Creating multi-page forms
+  - [About the schemaform](building-a-form/about-the-json-schema-standard-and-react-jsonschema-form.md)
```

Comment:
I'd call the library us-forms-system and come up with a different name for the form configuration object that is housed in forms.js that currently has the name `formConfig`. To make things more confusing, that object incorporates stuff from JSON Schema and react-jsonschema-form.

Reply:
> I'm not what the right title for this section is though, perhaps more descriptive, like About building forms using JSON Schema

Because this is a conceptual article, it ideally needs a noun. What about "About the US Forms System library", and then in the short description, I'll mention JSON Schema?